### PR TITLE
Add mentoring combo button to <Nudge />

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-project-management/issues/152.

### CODE REVIEW

File: app/javascript/components/student/solution-summary/MentoringComboButton.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          offset: [0, 14],
+        },
+      },
+    ],
+  })
+
+  const classNames = ['c-combo-button', className]
+
+  return (
+    /* TODO: Extract into a common component in the future */
```
Comment (ErikSchierboom) [inline app/javascript/components/student/solution-summary/MentoringComboButton.tsx:50]: Can this now be removed, or is this about certain child parts of the combo button?

File: app/javascript/components/student/solution-summary/MentoringComboButton.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  if (!portalContainer) {
+    throw new Error('No portal container found')
+  }
+
+  return ReactDOM.createPortal(
+    <div ref={setPanelElement} style={styles.popper} {...attributes.popper}>
+      {open ? children : null}
+    </div>,
+    portalContainer
+  )
```
Comment (ErikSchierboom) [inline app/javascript/components/student/solution-summary/MentoringComboButton.tsx:119]: Is this wrapping in a portal code maybe a candidate for a separate component?

File: test/system/components/student/solution_summary/mentoring_button_test.rb [hunk 3]
```diff
@@ -46,6 +46,47 @@ class SolutionSummarySectionTest < ApplicationSystemTestCase
           assert_text "Want to get mentored by a friend?"
         end
       end
+
+      test "shows discussions within nudge section" do
```
Comment (ErikSchierboom) [inline test/system/components/student/solution_summary/mentoring_button_test.rb:50]: Are we showing the discussions or just a link to the mentoring? Maybe something like:

```suggestion
      test "shows links to continue mentoring within nudge section" do
```

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
@kntsoriano Nice work! I'm going to merge this but see Erik's comments if you want to follow up in a seperate PR :)

**Pull request merged**